week-2/Homework/mandatory/2-function-creation.js

```diff
@@ -37,7 +55,16 @@ Write a function that:
 - numbers greater 100 must be replaced with 100
 */
 
-function formatPercentage(arr) {}
+function formatPercentage(arr) {
+  for (let i = 0; i < arr.length; i++) {
+    arr[i] = Number(arr[i].toFixed(2));
+    if (arr[i] > 100) {
+      arr[i] = 100;
+    }
+    arr[i] = arr[i] + "%";
+  }
+  return arr;
+}
 
```

Comment:
works as expected; 
would you like to give more explanation of how Number object works?